.gitignore

```diff
@@ -14,6 +14,8 @@ phpdoc
 # Composer
 /vendor/
 output
+# Allow the QM output directory, as it's needed.
+!query-monitor/output
```

Comment:
Would it be better to make the rule above more explicit? i.e. `/output/`? As it's likely we could run into similar problems with other paths in the future?

Reply:
Could do. 

`output` was [added](https://github.com/Automattic/vip-go-mu-plugins/commit/037bdde9e0c3506708576c6f2d0ba0ca1724246b) four years ago, when adding PHPDoc-generation, and I would presume that this is the first instance which has arisen from it.

Someone let me know (or just go ahead) if this change to `/output/` is definitely wanted.